samples/bot-calling-meeting/csharp/Source/CallingBotSample/Bots/MessageBot.cs

```diff
@@ -83,6 +89,31 @@ protected override async Task OnMessageActivityAsync(ITurnContext<IMessageActivi
             }
         }
 
+        /// <summary>
+        /// Called when a new user is added to the chat/channel, or when the application is first installed
+        /// We are using this to send the welcome card on first install
+        /// </summary>
+        /// <param name="teamsMembersAdded">Users added</param>
+        /// <param name="teamInfo"></param>
+        /// <param name="turnContext"></param>
+        /// <param name="cancellationToken"></param>
+        /// <returns></returns>
+        protected override async Task OnTeamsMembersAddedAsync(IList<TeamsChannelAccount> teamsMembersAdded, Microsoft.Bot.Schema.Teams.TeamInfo teamInfo, ITurnContext<IConversationUpdateActivity> turnContext, CancellationToken cancellationToken)
```

Comment:
This is something that I meant to add earlier, but writing docs it felt important to make it easier to explain how the entry point works. Most of the other changes in this file are because of an effort to reduce code reuse due to this addition.